client/List.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+class Item extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {isCompleted: false};
+        this.handleClick = this.handleClick.bind(this);
+    }
+    handleClick(){
+        this.setState({isCompleted: !this.state.isCompleted});
+    }
+    render(){
+        const {name} = this.props;
+        const itemWithStatus = this.state.isCompleted ? <strike>{name}</strike> : name; 
+        return (
+        <span onClick={this.handleClick}>
+            {itemWithStatus}
+        </span>
+        );
+    }
+};
+
+const List = (props) => {
+  const { items } = props;
+
+  return (
+    <ul>{items.map((item, index) => <li key={`item_${index}`}><Item name={item}/></li>)} </ul>
+  );
```

Comment:
Looks like you're using mixed indentation.  IMO stick with 4 spaces.  More readable.

Reply:
yea it's the switching from atom to vscode haha